### PR TITLE
[keras.WandbCallback] Flag to disable FLOPs calculation

### PR DESCRIPTION
**Pull request by ayulockin**
Fixes [WB-10339](https://wandb.atlassian.net/browse/WB-10339)

Description
-----------
Adds a `compute_flops` argument for the user to control if FLOPs calculation should be done or not.

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

File: wandb/integration/keras/keras.py [hunk 1]
```diff
@@ -698,11 +700,14 @@ def on_train_begin(self, logs=None):
                     f"Skipping logging validation data. Error: {str(e)}"
                 )
 
-        if _can_compute_flops():
+        if self.compute_flops and _can_compute_flops():
             try:
                 wandb.summary["GFLOPs"] = self.get_flops()
             except Exception as e:
-                wandb.termwarn("Unable to compute FLOPs for this model.")
```
Comment (dmitryduev) [inline wandb/integration/keras/keras.py:705]: I would suggest to simply add `repeat=False` here so that the term warning is only printed once.

Comment (ayulockin) [reply wandb/integration/keras/keras.py:706]: Hey @dmitryduev, the `get_flops` method is called just once `on_train_begin` thus, adding `repeat=False ` will not be helpful IMO. As per the discussion [here](https://wandb.atlassian.net/browse/WB-10339), we would like to fail it silently and mention it in the documentation.

File: wandb/integration/keras/keras.py [hunk 2]
```diff
@@ -702,8 +702,8 @@ def on_train_begin(self, logs=None):
         if _can_compute_flops():
             try:
                 wandb.summary["GFLOPs"] = self.get_flops()
-            except Exception as e:
-                wandb.termwarn("Unable to compute FLOPs for this model.")
+            except:
+                pass
```
Comment (ayulockin) [inline wandb/integration/keras/keras.py:706]: I don't think it's a good practice to use `except: pass` for failing something silently. If it's not, can someone suggest a better approach?

File: wandb/integration/keras/keras.py (hunk 1 above)
Comment (dmitryduev) [reply wandb/integration/keras/keras.py:706]: How about this report: https://github.com/wandb/wandb/issues/3916

Comment (ayulockin) [reply wandb/integration/keras/keras.py:706]: cc: @morganmcg1 what do you think?

Comment (morganmcg1) [reply wandb/integration/keras/keras.py:706]: @ayulockin I think `repeat=False` makes sense

Comment (dmitryduev) [reply wandb/integration/keras/keras.py:706]: @ayulockin any updates? do you want this in the upcoming release?

**Comment by ayulockin** [comment]
Not required anymore since flag is added in this PR: #4310

**Pull request closed without merging**